test/travis_commands.jl

```diff
@@ -1,34 +1,29 @@
 using Pkg
 
-Pkg.add("ImageMagick")
-Pkg.build("ImageMagick")
-
-# Pkg.clone("GR")
-# Pkg.build("GR")
-
-Pkg.checkout("https://github.com/JuliaPlots/PlotReferenceImages.jl.git")
-
-# Pkg.clone("https://github.com/JuliaStats/KernelDensity.jl.git")
+# need this to use Conda
+# ENV["PYTHON"] = ""
 
-# Pkg.add("StatPlots")
-Pkg.checkout("PlotUtils")
-Pkg.checkout("RecipesBase")
+Pkg.add([
+    PackageSpec("ImageMagick"),
+    # PackageSpec("GR"),
+    PackageSpec(url="https://github.com/tkf/PlotReferenceImages.jl", rev="pkg3"),
```

Comment:
Revert a5855929eec3dae71d4099173e93a25e3258adf7 once https://github.com/JuliaPlots/PlotReferenceImages.jl/pull/30 is merged